javascript/new-src/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "OLA",
+  "version": "0.9.4",
```

Comment:
This still needs to be populated via a Makefile .in file (or remove the version if that's easier).